### ISSUE TITLE
BinSkim warns that Microsoft.Web.WebView2.Core.dll does not enable Control-flow Enforcement Technology (CET) Shadow Stack mitigation

### ISSUE THREAD

**Issue by aschultz** [MicrosoftEdge/WebView2Feedback#3689]
**Description**
warning BA2025: 'Microsoft.Web.WebView2.Core.dll' does not enable the Control-flow Enforcement Technology (CET) Shadow Stack mitigation. To resolve this issue, pass /CETCOMPAT on the linker command lines

**Version**
microsoft.web.webview2.1.0.1901.177

**Regression**
No

**Repro Steps**
1. Download and unzip BinSkim (https://github.com/Microsoft/binskim) from NuGet
2. Run BinSkim against dlls

```
binskim analyze --sympath Cache*C:\SymCache;SRV*https://symweb;SRV*https://msdl.microsoft.com/download/symbols C:\temp\microsoft.web.webview2.1.0.1901.177\runtimes\win-x64\native_uap\Microsoft.Web.WebView2.Core.dll
```

Expectation:
No issues

[AB#45872653](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/45872653)

**Comment by champnic** [comment]
Thanks for the bug report @aschultz. I've added this bug to our backlog to take a look.

**Comment by champnic** [comment]
This should be fixed in runtime versions 119.0.2111.0+. Thanks!
